src/selectors/host/selectors.js

```diff
@@ -47,3 +47,6 @@ export const getHostStatusText = host => {
   const hostStatus = getHostStatus(host);
   return statusTextMap[hostStatus] ? statusTextMap[hostStatus] : hostStatus;
 };
+
+export const getHostRole = hostMachine =>
+  get(hostMachine, ['metadata', 'labels', 'machine.openshift.io/cluster-api-machine-role'], '-');
```

Comment:
returning `-` is this is very UI specific. Can you just return null and in `BaremetalHostRole` do 
```
getHostRole(machine) || DASHES
```

Reply:
Done